src/main/java/org/betonquest/betonquest/compatibility/heroes/HeroesAttributeCondition.java

```diff
@@ -0,0 +1,54 @@
+package org.betonquest.betonquest.compatibility.heroes;
+
+import com.herocraftonline.heroes.Heroes;
+import com.herocraftonline.heroes.attributes.AttributeType;
+import com.herocraftonline.heroes.characters.Hero;
+import org.betonquest.betonquest.Instruction;
+import org.betonquest.betonquest.VariableNumber;
+import org.betonquest.betonquest.api.Condition;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.betonquest.betonquest.exceptions.QuestRuntimeException;
+import org.betonquest.betonquest.id.ID;
+import org.betonquest.betonquest.utils.PlayerConverter;
+
+/**
+ * Checks an attribute of a player and if greater than or equal to a level
+ * Formatted as such "heroesattribute {attribute} {level}
```

Comment:
```suggestion
```
We dont document the syntax in the code, and there are other future planned for this. SO i suggest to just delete this